README.md

```diff
@@ -156,11 +157,18 @@ being accidentally leaked on GitHub.
 
 Also note that the AWS IAM limits sending
 to approved `from` addresses.
-Again, you can set that via environment variable (`FXA_EMAIL_SENDER`)
-or in `config/local.json`:
+Again, you can set that via environment variables:
+
+* `FXA_EMAIL_SENDER_ADDRESS`
+* `FXA_EMAIL_SENDER_NAME`
+
+Or in `config/local.json`:
 
 ```json
-  "sender": "Firefox Accounts <verification@latest.dev.lcip.org>",
+  "sender": {
+    "address": "verification@latest.dev.lcip.org",
+    "name": "Firefox Accounts"
```

Comment:
I separate the address and name parts of this setting so that it works better with the Sendgrid API.